.travis.yml

```diff
@@ -11,7 +11,12 @@ jobs:
     - name: “Python 3.8 make lint only”
       python: "3.8"
       install: pip install flake8
-      script: make lint
+      script:
+        - make lint
+        # Run all flake8 tests on modified code
+        - if [ "$TRAVIS_EVENT_TYPE" == "pull_request" ]; then
+            git diff origin/master -U0 | flake8 --diff;
```

Comment:
I'm a little nervous about checking _all_ flake8 errors; but I'm happy to try it to see how many of the open PRs break 😅 .